src/hotspot/share/prims/jvmti.xml

```diff
@@ -12892,15 +12892,21 @@ myInit() {
          id="ThreadStart" const="JVMTI_EVENT_THREAD_START" num="52" phase="start">
     <description>
       A thread start event is generated by a new thread before its initial
-      method executes. The capability
+      method executes.
+      <p/>
+      This event is generated by platform threads. It is also generated by
+      virtual threads when the capability
       <internallink id="jvmtiCapabilities.can_support_virtual_threads">
-      <code>can_support_virtual_threads</code></internallink> determines
-      if a new virtual thread generates a <code>ThreadStart</code> event or
-      a <eventlink id="VirtualThreadStart"/> event. If disabled, a virtual
-      thread generates a <code>ThreadStart</code> event. If enabled, a virtual
-      thread generates a <code>VirtualThreadStart</code> event.
+      <code>can_support_virtual_threads</code></internallink> is not enabled.
+      This ensures that agents that enable this event, but without this
+      capability, are notified when all threads start.
```

Comment:
It feels like this sentence can be improved. But I can't suggest anything yet.

Reply:
I did further word smithing and I hope the proposal text is clearer now.